src/systems/particle_emitter2/ParticleEmitter2.cc

```diff
@@ -100,20 +93,9 @@ bool ParticleEmitter2Private::EmittersService(
   _res.Clear();
 
   // Lock and wait for an iteration to be run and fill the state
```

Comment:
I think this comment no longer applies?

Reply:
Removed in d57cfa204f4f713031e844edda0a6f003940f60b